src/validations/rules/sar.sch

```diff
@@ -382,6 +382,36 @@
                 test="oscal:attestation[oscal:part[@name = 'authorization-statements']/oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'recommend-authorization']]">
                 There must exist an attestation with a part containing a property with a name of 'recommend-authorization'.</sch:assert>
 
+            <sch:let
+                name="pl-2-other-than-satisfied-findings"
+                value="oscal:finding/oscal:target[@type = 'objective-id'][oscal:status ne 'satisfied'][matches(@target-id, '^pl-2\.')]/@target-id" />
+            <sch:report test="true()">PL-2: <sch:value-of select="$pl-2-other-than-satisfied-findings"/></sch:report>
```

Comment:
This looks like debugging code.

Reply:
shoot.  Yes it is.  Thanks for the catch.